src/components/userNetwork/Network.js

```diff
@@ -87,26 +108,73 @@ const Network = () => {
           onButtonClick={handleFindUser}
           onResetSearch={handleRefreshSearch}
         />
+        <br />
+
+        <Typography
+          style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+        >
+          Filters
+        </Typography>
+        <DropdownFilter
+          placeholder="Select a Year and Semester"
```

Comment:
Can you possibly make this placeholder text smaller and gray also? Also is the styling matching the search bar styling, or is Saf covering that later?

Reply:
I'll cover styling